### PR TITLE
feat: expand block tests

### PR DESCRIPTION
**Pull request by valfaro123**
### 🔥 Summary
- We want to expand our block page cypress tests for #372 

### 🤓 Solution
- Update to test flow on blocks page

### CODE REVIEW

File: app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx [hunk 1]
```diff
@@ -72,10 +72,7 @@ export const BlockDetailsCard: React.FC<BlockDetailsCardProps> = ({
           <DetailDataRowWrapper>
             <li>
               <DetailDataLabel>{t('block-height')}</DetailDataLabel>
-              <DetailDataValue
-                data-testid="block-height"
-                data-cy="block-height"
-                isLargeText>
+              <DetailDataValue data-testid="block-height" isLargeText>
```
Comment (DorianKwan) [inline app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx]: Why are we removing the block height here?

File: app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx [hunk 2]
```diff
@@ -267,15 +265,17 @@ const DetailDataRowWrapper = styled.ul`
   display: flex;
   flex-direction: column;
 
-  @media only screen and (min-width: ${breakpoints.lg}) {
+  @media only screen and (min-width: ${defaultTheme.typography.breakpoints
+      .lg}) {
```
Comment (DorianKwan) [inline app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx]: We should keep this as it was.  Prettier and eslint are arguing here

File: app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx (hunk 1 above)
Comment (DorianKwan) [reply app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx]: Removing this should break the test

File: test/cypress/e2e/block.cy.ts [hunk 3]
```diff
@@ -1,12 +1,93 @@
...
-    const blockHeight = 1;
+  beforeEach(() => {
+    cy.visit(`/block/${blockHash}`);
+  });
 
-    cy.visit(`/block/${blockHeight}?type=height`);
+  const rawDataHeading = 'AccordionItemHeading';
+  const rawDataButton = 'AccordionItemButton';
 
+  it('can visit at /block/:blockHeight', () => {
```
Comment (DorianKwan) [inline test/cypress/e2e/block.cy.ts]: Can you add tests for both mobile screen size and desktop.  It's more important for this test as there are two largely different UI's base on the screensize

**Review by DorianKwan — CHANGES_REQUESTED** [review]

File: app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx (hunk 1 above)
Comment (valfaro123) [reply app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx]: This was a mixup when merging branches. Fixing this.

**Review by DorianKwan — APPROVED** [review]

**Pull request merged**